config_ext_test.go

```diff
@@ -0,0 +1,350 @@
+package config_test
+
+import (
+	"strings"
+	"testing"
+
+	. "go.uber.org/config"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIntegration(t *testing.T) {
+	first := strings.NewReader(`
```

Comment:
is it worth using the yaml pointers to make sure they work as expected?

Reply:
Good point, definitely worthwhile.